add badge and t4f affiliation to readme (#11)
ProfessionalsAgainstGenocide/verifiedPetition: README.md

signing the petition how signatures were collected. Please feel free to open issues if
you discover ways to add duplicate emails, skip captcha, or otherwise spam the service.

> `verifiedPetition` is an official project of <a target="_blank" href="https://techforpalestine.org/">Tech for Palestine</a>.

Reviewer: I think @pbiggar might prefer we call this a "community experiment" for now. I'll need to talk to Tom Hall to make it an official project

Author: My impression was that Paul considered unvetted projects as "community" but now that he's spoken with you he considers this official but I might be misunderstanding. Let's give him a chance to take a look.

Reviewer: Yeah... during our conversation when we transferred the project to T4P's GitHub, there was a quick monologue where @pbiggar pondered if it can be included in the org before becoming an official project.

We should definitely give him a chance to weigh in though

Reviewer: I pinged the discord channel to try and get some eyes

Reviewer: From my conversation on discord: 

> Sorry for the hold-up but we're in the middle of formalizing our process around creating official projects, so I'll have to confer with the person who is managing that.

I think it might be quickest to just do this

```suggestion
`verifiedPetition` is an community project associated with <a target="_blank" href="https://techforpalestine.org/">Tech for Palestine</a>.
```

and come back and change it once we’re official. I think they’re working on some standard (being on their domain, etc, some standard for where to host, etc) for considering a project official

Author: That makes sense, I'll make that change this afternoon (busy morning).
